vendor.conf

```diff
@@ -101,3 +102,35 @@ k8s.io/client-go kubernetes-1.11.2
 k8s.io/kube-openapi d8ea2fe547a448256204cfc68dfee7b26c720acb
 k8s.io/kubernetes v1.11.2
 vbom.ml/util 256737ac55c46798123f754ab7d2c784e2c71783
+
+#get libnetwork packages
+github.com/docker/libnetwork 1a06131fb8a047d919f7deaf02a4c414d7884b83
```

Comment:
curious why all these dependencies are now needed on the client side 🤔

Reply:
Yeah, I was bummed this pulled in so much but it comes from the code just below this comment block:  https://github.com/docker/cli/pull/1721/files#diff-04999fb6f01d1478d54e2713a0b41a15R113

Reply:
I also noticed here that we're pulling in the backend; https://github.com/docker/cli/blob/99c7e4210fa49f65f82cacf7e5f728d7a2e247c0/vendor/github.com/docker/stacks/pkg/interfaces/interfaces.go#L6-L37

Many of these should not be in the CLI (and might not even work on macOS, Windows)

Reply:
If those dependencies pop up in the client code (libnetwork, hashcorp/memberlist, consul, etcd ?) it means the packages in `docker/stack` are not loosely coupled. There should be no reason for the `cli` to pull those dependencies :sweat: 

A small package split / reorganization on `docker/stack` is required for this to get in `docker/cli`

Reply:
Agreed, the `stacks/pkg/interfaces` package is meant for use only by components that are internal to the engine. We'll try to unwrangle why it's being referenced here